docs/source/onnxruntime/modeling_ort.mdx

```diff
@@ -66,19 +64,16 @@ from the Hub and converts it to an optimum onnxruntime model and pushes it back
 >>> from transformers import AutoTokenizer
 >>> from optimum.onnxruntime import ORTModelForSequenceClassification
 
-# load model from hub and convert
->>> model = ORTModelForSequenceClassification.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english",from_transformers=True)
+# Load model from hub and convert
+>>> model = ORTModelForSequenceClassification.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english", from_transformers=True)
 >>> tokenizer = AutoTokenizer.from_pretrained("distilbert-base-uncased-finetuned-sst-2-english")
 
-# save converted model
+# Save converted model
 >>> model.save_pretrained("a_local_path_for_convert_onnx_model")
 >>> tokenizer.save_pretrained("a_local_path_for_convert_onnx_model")
 
-# push model onnx model to HF Hub
->>> model.push_to_hub("a_local_path_for_convert_onnx_model",
-                  repository_id="my-onnx-repo",
-                  use_auth_token=True
-                  )
+# Push model onnx model to HF Hub
+>>> model.push_to_hub("a_local_path_for_convert_onnx_model", repository_id="my-onnx-repo", use_auth_token=True)
 ```
 
```

Comment:
we should include an example for converting Seq2Seq model here, even if it is the same code it would be good.